src/components/HomePageContent/LandingSection.tsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
```

Comment:
one merge, could you update the picture of https://openattestation.com/

Reply:
yup sure